lib/descriptor.js

```diff
@@ -24,3 +26,67 @@ const generateDescriptor = async (data, dpName = "scratchpad") => {
   return descriptor
 }
 module.exports.generateDescriptor = generateDescriptor
+
+const generateDescriptorForLocalFiles = function (dpName = "scratchpad") { 
+  return new Promise(function(resolve, reject) { 
+    fs.readdir('./', (err, files) => {
+      var descriptor = {
+          name: dpName,
+          resources: []
+      };
+      files.forEach((file) => {
+        if (path.extname(file) == ".csv") {
+          var name = path.basename(file, '.csv').replace(/ /g, "-").toLowerCase();
+          fs.readFile(file, (err, data) => {
+            if(err) throw reject(err)
+            parse(data, (error, values) => {
```

Comment:
Why are not we using `generateDescriptor()` function from above to do all this  (from line 42-54). it's kind of doing the same right? think we are duplicating ourselfs